tests/test_stream.py

```diff
@@ -540,7 +540,6 @@ def testUBXITERATE_ERR2(
         i = 0
         for raw, parsed in ubr:
             if not isinstance(parsed, str):

```

Comment:
good spot, but I've actually been a bit lazy with this particular test case and should have validated the stdout output rather than just count the lines. I've since updated this in the master branch.